src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/ApproveOrderService.java

```diff
@@ -67,4 +88,50 @@ public JudgeApproval resolveJudgeApproval(DraftOrdersWrapper draftOrdersWrapper,
             default -> null;
         };
     }
+
+    void buildConfirmationBody(FinremCaseDetails caseDetails, DraftOrdersWrapper draftOrdersWrapper) {
+
+        final List<String> ordersApproved = new ArrayList<>();
+        final List<String> ordersRepresentativeChanges = new ArrayList<>();
+        final List<String> ordersChanged = new ArrayList<>();
+        final List<String> ordersReviewLater = new ArrayList<>();
+
+        for (int i = 1; i <= 5; i++) {
+            JudgeApproval judgeApproval = resolveJudgeApproval(draftOrdersWrapper, i);
+
+            if (judgeApproval != null) {
```

Comment:
```
            ofNullable(judgeApproval).map(JudgeApproval::getDocument)
                .ifPresent(targetDoc -> captureFilenames());
```
Move the following logics (line 103 to 109) into a new method `captureFilenames`